### ISSUE TITLE
maximumNumberOfLogFiles & maximumFileSize

### ISSUE THREAD

**Issue by tianliangyihou** [CocoaLumberjack/CocoaLumberjack#1046]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](https://github.com/CocoaLumberjack/CocoaLumberjack#documentation) or [CocoaDocs](cocoadocs.org/docsets/CocoaLumberjack)
* [ ] I have searched for a similar issue in the [project](https://github.com/CocoaLumberjack/CocoaLumberjack/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           |  ios 
 Platform Version        | 11.3.1
 CocoaLumberjack Version | 3.5.0
 Xcode Version           |  Xcode 10.1
 Repro rate              | all the time (100%) 

### Issue Description and Steps
when I write
```
    Logger *fileLogger;
    fileLogger = [[DDFileLogger alloc] init]; // File Logger
    fileLogger.rollingFrequency = 60 * 60 * 24; // 24 hour rolling
    fileLogger.logFileManager.maximumNumberOfLogFiles = 3;
    fileLogger.maximumFileSize =  100 * 1024;
    for (int i = 0 ; i < 10000000; i++) {
        DDLogInfo(@"fileLogger.maximumFileSize = 1 * 1024 * 1024;");
    }
```
But program only writes in one file without restriction on size.

**Comment by ffried** [comment]
This is the same underlying issue as #1041 and is fixed in #1042...

**Comment by tianliangyihou** [comment]
Thank you very much for your reply. I have understanded.

**Comment by sushichop** [comment]
Fixed by #1042 .
